tests/migration_test.go

```diff
@@ -624,12 +625,12 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 
 			assertConnectivityToService := func(msg string) {
 				By(msg)
-				job := tests.NewHelloWorldJobTCP(fmt.Sprintf("%s.%s", hostname, subdomain), strconv.FormatInt(int64(port), 10))
+				job := network.NewHelloWorldJobTCP(fmt.Sprintf("%s.%s", hostname, subdomain), strconv.FormatInt(int64(port), 10))
```

Comment:
@EdDev 
are we ok with using network package here ?
otherwise we can / should move this specific Context (subdomain) to another file under network folder

Reply:
We should not import the tests into other tests.

Just move it to `libnet`.

Reply:
Thanks, done
hopefully done correct, had to split like this in order to avoid cycle import

note that we can combine the two job.go files into one that exists on libnet if desired